cmd/snap-bootstrap/initramfs_systemd_mount_test.go

```diff
@@ -52,7 +53,7 @@ exit 1
 `)
 	defer cmd.Restore()
 
-	err := main.DoSystemdMount("something", "somewhere only we know", nil)
+	err := main.DoSystemdMount("something", "somewhereonlyweknow", nil)
```

Comment:
do we need to change this?

Reply:
So now not really, I had to modify it because I also applied the restrictions to the where argument so it returned an error earlier due to that. I just reverted that particular change.